File: HW 11/Python/main.py
from floodfill import floodFill

def main():
    image = [[1,1,1],
             [1,1,0],
             [1,0,1]]
    sr, sc, color = 1, 1, 2
    modified_image = floodFill(image, sr, sc, color)
    print(modified_image) #output: [[2, 2, 2], [2, 2, 0], [2, 0, 1]]

if __name__ == "__main__":
    main()

File: HW 11/Java/Main.java
public class Main {
    public static void main(String[] args) {
        FloodFill floodFill = new FloodFill();
        int[][] image = {
            {1, 1, 1},
            {1, 1, 0},
            {1, 0, 1}
        };
        int sr = 1, sc = 1, color = 2;
        int[][] modifiedImage = floodFill.floodFill(image, sr, sc, color);
        for (int[] row : modifiedImage) {
            for (int pixel : row) {
                System.out.print(pixel + " ");
            }
            System.out.println();
        }
    }
}
